rollup.config.js

```diff
@@ -27,6 +27,7 @@ const appConfiguration = {
   },
   plugins: [
     svelte({
+      hydratable: true,
```

Comment:
(see my comment in `server.js` and `src/app.main.ts`)

```suggestion
```

Reply:
same as above